osu.Game.Tests/Visual/SongSelect/TestSceneBeatmapCarousel.cs

```diff
@@ -834,40 +842,35 @@ private BeatmapSetInfo createTestBeatmapSet(int id)
                     Title = $"test set #{id}!",
                     AuthorString = string.Concat(Enumerable.Repeat((char)('z' - Math.Min(25, id - 1)), 5))
                 },
-                Beatmaps = new List<BeatmapInfo>(new[]
+                Beatmaps = getBeatmaps(randomDifficultyCount ? RNG.Next(1, 20) : 3).ToList()
+            };
+        }
+
+        private IEnumerable<BeatmapInfo> getBeatmaps(int count)
+        {
+            int id = 0;
+
+            for (int i = 0; i < count; i++)
+            {
+                float diff = (float)i / count * 10;
+
+                string version = "Normal";
+                if (diff > 6.6)
+                    version = "Insane";
+                else if (diff > 3.3)
+                    version = "Hard";
```

Comment:
I don't think it can, since they don't match. Some of the tests rely on specific strings and would need to be updated, wasn't really wanting to do so.